xAPI-Communication.md

```diff
@@ -1118,7 +1113,7 @@ about the Agent it received in the request.
 	<tr>
 		<td>objectType</td>
 		<td>String</td>
-		<td>"Person"</td>
+		<td>Person</td>
```

Comment:
We use quotes elsewhere to denote that the value of a property is fixed in the scenario being described. e.g. https://github.com/adlnet/xAPI-Spec/blob/1.0.3-Communication-Edits/xAPI-Data.md#details-2